Dockerfile

```diff
@@ -4,6 +4,10 @@ LABEL version="2.1"
 LABEL description="LDAP Team Sync for GitHub"
 LABEL maintainer="GitHub Services <services@github.com>"
 
+ARG TZ='EST5EDT'
```

Comment:
Would UTC make sense here as the default?

Reply:
```suggestion
ARG TZ='UTC'
```